tests/kernel/install_ltp.pm

```diff
@@ -224,12 +224,24 @@ sub add_ltp_repo {
     zypper_ar($repo, name => 'ltp_repo');
 }
 
-sub install_from_repo {
-    my $pkg = get_var('LTP_PKG', (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp');
+sub get_default_pkg {
+    return (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp';
```

Comment:
it doesnt seem you need `is_sle`. `want_stable` default contains `is_sle`. i havent found an example with LTP_STABLE to get the idea when the variable is set but it is not sle. Can you provide to me such example?

Reply:
`is_sle` and `want_stable` is not relevant to this PR. Believe me it's needed (setup for QAM tests; not everything is documented). It'd help me more if you could test it for cloud (or tell me how I can do it), but that can wait till final code.

Reply:
@b10n1k `want_stable` can be used on o3. But stable LTP on o3 is also called `ltp`. 
`qa_test_ltp` is 1) only on osd 2) is stable. That's why double check (for me it was kind of documentation that's why I had no problem to use it twice).

Reply:
@mloviska FYI

Reply:
@mdoucha FYI

Reply:
ok. just to make clear. can LTP_STABLE be ever true on o3? 

i agree that it is not relevant to this PR and thats why we can close this conversation. But if it a small glitch it would be also an easy fix as you touch that particular code. or i would like to know for the future. no blocking issue

Reply:
It should not do anything since https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12203. Regardless setting it you always get `ltp` package with master branch. I'm open to suggestions what to do with the variable, or just send PR. 
It's just instead of tuning LTP openQA runner we prefer to work on LTP itself (openQA is just a runner, tests are what brings a benefit; unless some significant improvement like speedup). But I'll try to have look and improve openQA LTP docs.

Reply:
thanks @pevik